generate_clients.py

```diff
@@ -147,4 +131,4 @@ def _setup_logger(log_file):
     log = os.path.abspath(flags.log)
     user_config = os.path.abspath(os.path.expanduser(flags.user_config)) if flags.user_config else None
 
-    generate_clients(root_dir, log, user_config)
+    generate_clients(root_dir, flags.languages.split(), flags.artman_config, log, user_config)
```

Comment:
And then if nargs was used for languages, we don't have to pass in a quoted list of space-separated lang values that gets `.split()`

Reply:
Done.